gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/OrganizationLicenseRepository.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.management.api;
+
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.repository.management.model.OrganizationLicense;
+import java.util.Optional;
+
+public interface OrganizationLicenseRepository extends CrudRepository<OrganizationLicense, String> {
+    Optional<OrganizationLicense> findByOrganizationId(String organizationId) throws TechnicalException;
```

Comment:
Since the organizationId is the id, you already have a findById in the CrudRepository interface.

Reply:
I will be adding an id and  a reference id + type coupling, so I'll keep this method as it is.